tests/conftest.py

```diff
@@ -272,6 +272,17 @@ def influxdb(base_influxdb):
     base_influxdb.reset()
 
 
+@pytest.fixture(scope="session")
+def argo_version(repo_dir):
+    with open(os.path.join(repo_dir, "Dockerfile-GordoDeploy")) as f:
+        match = next(re.finditer(r'ARGO_VERSION="(\w\d+.\d+.\d+)"', f.read()), None)
```

Comment:
> Some people, when confronted with a problem, think "I know, I'll use regular expressions." Now they have two problems.

Just kidding. Love it. Or at least, I accept it and will let it live in my house.

Reply:
haha, I've heard that. :laughing:  I've also heard Facebook doesn't allow regex in their code base, but I'm not certain how true that is.